core/src/main/java/com/scalar/db/storage/cassandra/CassandraConfig.java

```diff
@@ -0,0 +1,6 @@
+package com.scalar.db.storage.cassandra;
+
+public class CassandraConfig {
```

Comment:
In v3.x, there isn't `CassandraConfig`, but I added it to provide `STORAGE_NAME`, as the same as other storages.